source/uwp/SharedVisualizer/ViewModel/DocumentViewModel.cs

```diff
@@ -96,6 +102,9 @@ protected override async void LoadPayload(string payload)
                         _renderedAdaptiveCard.Action += async (sender, e) =>
                         {
                             var m_actionDialog = new ContentDialog();
+#if USE_WINUI3
+                            m_actionDialog.XamlRoot = this.MainPageViewModel._root;
```

Comment:
Root is ambiguous and can mean the root UI element for example:)
```suggestion
                            m_actionDialog.XamlRoot = this.MainPageViewModel._xamlRoot;
```

Isn't there a way to get the root during the runtime?

Reply:
I couldn't figure out any other way to get the XamlRoot from the ViewModel. Does the ViewModel have something that knows about the xaml tree that I missed?

Reply:
Nope, you're probably right..